standalone/test/scripts/parse_dump.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
```

Comment:
This is needed?

Reply:
Maybe rewrite in C++ in order to be able to dump this on unsuccessfull tests

Reply:
You can directly map header id to color btw

Reply:
Consider using `printDump` in standalone/test/unit_tests/data_conversion_layer/unittest_monitoring_frame_serialization_deserialization.cpp

Reply:
@rfeistenauer ?